deploy/cr/expose/cr_ingress.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: infinispan.org/v1
+kind: Infinispan
+metadata:
+  name: example-infinispan
+spec:
+  replicas: 2
+  expose:
+    type: Ingress
```

Comment:
@rigazilla please change to Route as you've removed this expose type

Reply:
yeah thanks @dmvolod, I've removed it, since seems a duplicate of route.yaml

Reply:
Yes, sure
@rigazilla also we need to update doc/design.adoc with actual model. I will try to take care about it.

Reply:
thanks!